src/engine/v3/V3Interpreter.v3

```diff
@@ -192,7 +205,8 @@ class V3Interpreter extends WasmStack {
 				frame.pc = pc = codeptr.pos; // update pc after decoding locals
 				// Fire entry probe(s).
 				if (func.decl.entry_probed) {
-					var throwable = Instrumentation.fireLocalProbes(DynamicLoc(func, 0, TargetFrame(frame)));
+					// var throwable = Instrumentation.fireLocalProbes(DynamicLoc(func, 0, TargetFrame(frame)));
```

Comment:
I had to kill every line in V3Interpreter that used `TargetFrame`, since it's a different type depending on which target it's compiled with. Not sure what the best long term solution is.